### PR TITLE
Feature/player age

### PR DESCRIPTION
**Pull request by andredavii**
player age function

### CODE REVIEW

**Review by Zwilson1301 — APPROVED** [review]
Looks good to me.

File: source/CardGames.py [hunk 1]
```diff
@@ -156,3 +156,27 @@ def dealCards(self, numCards: int, players: "list[Player]"):
...
+    # Show each player's hand
+    for player in players.values():
+        print(f'{player.name}:')
+        player.showHand(True)
+        print()
+
+def player_ages(players):
+    playerAge = []
+    for player in players.values():
+        while True:
```
Comment (kerenvr) [inline source/CardGames.py:170]: Do you need the while loop?

Comment (andredavii) [reply source/CardGames.py:170]: Good point. I will try experimenting without it and see If the code still runs. Thanks for the feedback!

File: source/CardGames.py [hunk 2]
```diff
@@ -156,3 +156,27 @@ def dealCards(self, numCards: int, players: "list[Player]"):
...
+            try:
+                age = int(input("Enter {}'s age: ".format(player.name)))
+                playerAge.append((player, age))
+                break
+            except ValueError:
```
Comment (RobertWheel) [inline source/CardGames.py:175]: Great use of the except value; it improves the codes robustness, code looks good, great job

**Pull request merged**